packages/stargate/src/stargateclient.ts

```diff
@@ -22,6 +23,45 @@ export interface SequenceResponse {
   readonly sequence: number;
 }
 
+export interface BroadcastTxFailure {
+  readonly height?: number;
+  readonly code: number;
+  readonly transactionHash: string;
+  readonly rawLog?: string;
+  readonly data?: Uint8Array;
+}
+
+export interface BroadcastTxSuccess {
+  readonly height?: number;
+  readonly transactionHash: string;
+  readonly logs: JsonCompatibleValue;
```

Comment:
I tried this type for logs before, which ended up with big trouble. Since we create this field, I see two approaches:
1. Remove and just provide `rawLog` (for now). We can parse in test code.
2. Create a log parser that returns a checked and typed result

I'd start with 1. and create a ticket for 2.

Reply:
https://github.com/CosmWasm/cosmjs/issues/376